include/ftxui/screen/color.hpp

```diff
@@ -322,6 +323,20 @@ class Color {
   uint8_t blue_ = 0;
 };
 
+inline namespace literals {
+
+/// @brief Creates a color from a combined hex RGB representation, e.g. 0x808000_rgb
+inline Color operator""_rgb(unsigned long long int combined)
+{
+    assert(combined <= 0xffffffU);
+    auto const red = static_cast<uint8_t>(combined >> 16);
+    auto const green = static_cast<uint8_t>(combined >> 8);
+    auto const blue = static_cast<uint8_t>(combined);
+    return Color(red, green, blue);
+}
+
+}
+
 }  // namespace ftxui
 
 #endif /* end of include guard: FTXUI_COLOR_H_ */
```

Comment:
Could you add an example in the ./example/dom/ directory?

Reply:
I amended the `style_color.cpp` example, is this sufficient?